languages/en.php

```diff
@@ -34,6 +34,9 @@
 	'event_calendar:day_label' => "Day",
 	'event_calendar:week_label' => "Week",
 	'event_calendar:month_label' => "Month",
+	'event_calendar:day_label_with_separator' => "Day: ",
+	'event_calendar:week_label_with_separator' => "Week: ",
+	'event_calendar:month_label_with_separator' => "Month: ",
```

Comment:
The date string should also be included in the translation:
```php
'event_calendar:month_label_with_separator' => "Month: %s",
```
There are languages that are read from right to left, so then the translation might be:
```php
'event_calendar:month_label_with_separator' => "%s شهر",
```
(Just an example. I don't know the correct Arabic format.)